packages/devextreme/js/__internal/ui/list/m_list.edit.decorator.ts

```diff
@@ -58,69 +86,79 @@ class EditDecorator extends (Class.inherit({}) as new() => {}) {
 
     clearTimeout(this._list._inkRippleTimer);
 
-    this._swipeStartHandler($itemElement, e);
+    this._swipeStartHandler($itemElement);
   }
 
-  _itemSwipeUpdateHandler(e): void {
-    const $itemElement = $(e.currentTarget);
-
-    this._swipeUpdateHandler($itemElement, e);
+  _itemSwipeUpdateHandler(e: SwipeUpdateEvent['event']): void {
```

Comment:
It should be usual DxEvent<>. I am not sure that Swipe typing is appropriate here.